pontoon/uxactionlog/migrations/0002_update_locale_code.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.11 on 2024-07-24 18:49
+
+from django.db import migrations
+
+
+def update_locale_code(apps, schema_editor):
+    UXActionLog = apps.get_model("uxactionlog", "UXActionLog")
+
+    for log in UXActionLog.objects.filter(action_type="LLM Dropdown Select"):
+        data = log.data
+        if "targetLanguage" in data:
+            data["localeCode"] = data.pop("targetLanguage")
+            log.data = data
+            log.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("uxactionlog", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.RunPython(update_locale_code),
+    ]
```

Comment:
We should make the migration [reversible](https://docs.djangoproject.com/en/5.0/ref/migration-operations/#runpython). Example:
https://github.com/mozilla/pontoon/blob/main/pontoon/base/migrations/0064_populate_unique_id.py#L23